src/nmodl/solve.cpp

```diff
@@ -155,7 +155,7 @@ void solvhandler() {
         switch (fun->subtype) {
         case DERF:
             if (method == SYM0) {
-                method = lookup("adrunge");
+                method = lookup("derivimplicit");
```

Comment:
@alkino : do we know what this means? i.e. replacing here adrunge with derivimplicit is "ok" ?

Reply:
I asked to @nrnhines: https://github.com/neuronsimulator/nrn/issues/2018#issuecomment-1286593424

Reply:
@nrnhines : I assume `method == SYM0` means user hasn't specified SOLVE method. In that case, what should be the fallback instead of adrunge now?

Reply:
Sorry for noise - "ignore" my "ignorance" 😒   (I started reviewing code without reading previous email notifications)